### PR TITLE
Telegraf manual plugins

### LINKED ISSUE #845: Add example for manually adding Telegraf plugins
We need to walk users through the process of:

1. Find the plugin you want to enable (plugins list)
2. Visit the plugin page on GH
3. Copy and paste the example config into your `telegraf.conf`
4. Restart Telegraf

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/write-data/use-telegraf/manual-config/

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #845 

- Added link from telegraf autoconfig page about adding other plugins. 
- Added steps to manual config page specifically detailing how to manually add plugins.
- Called out MQTT as an example in screenshot and tagged as keyword (per Tim's request).

### CODE REVIEW

File: content/v2.0/write-data/use-telegraf/auto-config/_index.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ for using Telegraf with InfluxDB v2.0._
    Plugins listed with a <span style="color:#32B08C">{{< icon "check" >}}</span>
    require no additional configuration.
    To configure a plugin or access plugin documentation, click the plugin name.
+
+   {{% note %}}
+   Not all available plugins are listed on this screen. For more information on manually configuring additional plugins, see [Manually add Telegraf plugins](/v2.0/write-data/use-telegraf/manual-config/).
```
Comment (noramullen1) [inline content/v2.0/write-data/use-telegraf/auto-config/_index.md:47]: @sanderson , wasn't sure if I should use this link or the list of plugins link here. Both seemed clunky? What do you think?

File: content/v2.0/write-data/use-telegraf/manual-config.md [hunk 2]
```diff
@@ -29,7 +30,16 @@ Output plugins define destinations where metrics are sent.
...
 
-## Enable and configure the InfluxDB v2 output plugin
+### Manually add Telegraf plugins
+
+To manually add any of the available Telegraf plugins [Telegraf plugins](/v2.0/reference/telegraf-plugins/), follow the steps below.
+
+1. Find the plugin you want to enable from the complete list of available [Telegraf plugins](/v2.0/reference/telegraf-plugins/).
+2. Click **View** to the right of the plugin name to open the plugin page on GitHub, as shown in the example with the MQTT plugin below.
+{{< img-hd src="/img/mqtt-plugin.png" alt="View in Github" />}}
+3. Copy and paste the example configuration into your Telegraf configuration file (typically named `telegraf.conf`).
```
Comment (noramullen1) [inline content/v2.0/write-data/use-telegraf/manual-config.md:40]: Not sure if I need to get into more detail with the example here?

File: content/v2.0/write-data/use-telegraf/auto-config/_index.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/write-data/use-telegraf/auto-config/_index.md:47]: I think this is fine.

File: content/v2.0/write-data/use-telegraf/manual-config.md [hunk 3]
```diff
@@ -29,7 +30,16 @@ Output plugins define destinations where metrics are sent.
...
 _See [Telegraf plugins](/v2.0/reference/telegraf-plugins/) for a complete list of available plugins._
 
-## Enable and configure the InfluxDB v2 output plugin
+### Manually add Telegraf plugins
+
+To manually add any of the available Telegraf plugins [Telegraf plugins](/v2.0/reference/telegraf-plugins/), follow the steps below.
+
+1. Find the plugin you want to enable from the complete list of available [Telegraf plugins](/v2.0/reference/telegraf-plugins/).
+2. Click **View** to the right of the plugin name to open the plugin page on GitHub, as shown in the example with the MQTT plugin below.
+{{< img-hd src="/img/mqtt-plugin.png" alt="View in Github" />}}
```
Comment (sanderson) [inline content/v2.0/write-data/use-telegraf/manual-config.md:39]: I'm not 100% a screenshot is need here.

**Pull request merged**